frontend/src/services/auth/auth.api.ts

```diff
@@ -1,7 +1,6 @@
 import { IUser } from '../../common/interfaces/user.interface';
-// import API from '../api';
+import API from '../api';
 
 export const login = async (): Promise<IUser> => {
-  // return API.post('auth/login').then((res) => res.data);
-  return Promise.resolve({ id: 'hahaha', name: 'Andrei Milioane' });
+  return API.post('auth/login').then((res) => res.data);
```

Comment:
De ce avem modificarea asta aici?

Reply:
A fost o modificare folosita la testare care a ramas in momentul in care s-a facut merge cu branch-ul respectiv.